Update cfwhile.json

scallahan-dev opened the pull request:
Added support for ColdFusion for the script syntax as well as a script syntax example.

scallahan-dev committed:
Update cfwhile.json

Added support for ColdFusion for the script syntax as well as a script syntax example.

pfreitag commented:
Hi @scallahan-dev thanks for your contribution. I would prefer to keep the coldfusion key out of the engines struct in this case since the tag is not supported on CF the omission implies this and it will display a message. 

Would love to have your cfscript while loop example in there though, so if you want to update the PR to include just that, I'd gladly accept that.

scallahan-dev commented:
@pfreitag That makes since, but the notice on the page is confusing and doesn't indicate anywhere that the script syntax is valid for ColdFusion. I'm open to ideas on how to make this more apparent.

pfreitag commented:
Yes, good point -- maybe in your example you can mention in the description of your example that the script syntax works on both Adobe ColdFusion and Lucee

scallahan-dev committed:
Update cfwhile.json

Script syntax with break usage and include a note of compatibility with ColdFusion.

scallahan-dev commented:
@pfreitag New commit with your requested changes.

scallahan-dev commented:
@pfreitag Just wanted to see if there were any other changes you wanted made to this PR.

pfreitag commented:
Thanks
